### ISSUE TITLE
Translatable

### ISSUE THREAD

**Issue by mondlicht-und-sterne** [cliambrown/PillTime#1]
Is this app translatable ?

**Comment by cliambrown** [comment]
@mondlicht-und-sterne Yes, it could be translated. All of the text is in [strings.xml](https://github.com/cliambrown/PillTime/blob/main/app/src/main/res/values/strings.xml), so someone could just create a copy of that file for each new language.

https://developer.android.com/guide/topics/resources/localization

**Comment by mondlicht-und-sterne** [comment]
@cliambrown 

Thank you, then I can translate it into German.
However, as I cannot do this pull request, I am attaching the translated file.

**Comment by cliambrown** [comment]
@mondlicht-und-sterne Thank you! Unfortunately I don't see a file attached to your message.

**Comment by mondlicht-und-sterne** [comment]
@cliambrown 

Yes, I know, I didn't quite finish - only up to almost 98 %.
Maybe I had another question yesterday and then fell asleep...

**Comment by mondlicht-und-sterne** [comment]
@cliambrown 

19     <string name="on">on</string>
23     <string name="at">at</string>

I have one more question:
What is meant by ‘on’ and ‘at’ in these 2 strings?

**Comment by cliambrown** [comment]
@mondlicht-und-sterne Here is an example sentence: Dose expires **on** DD/MM/YYYY **at** HH:MM

I've added a couple of comments to [strings.xml](https://github.com/cliambrown/PillTime/blob/main/app/src/main/res/values/strings.xml) in case that helps.

**Comment by cliambrown** [comment]
@mondlicht-und-sterne No pressure at all, but I just remembered that for the app to show up for German-speaking users, I would also need to supply translations of [short_description.txt](https://github.com/cliambrown/PillTime/blob/main/fastlane/metadata/android/en-US/short_description.txt) and [full_description.txt](https://github.com/cliambrown/PillTime/blob/main/fastlane/metadata/android/en-US/full_description.txt) as well.

**Comment by grego9** [comment]
Hi. I would like to contribute with the Spanish translation if that's ok.
